openhtf/__init__.py

```diff
@@ -217,8 +218,17 @@ def execute(self, test_start=None):
       self._test_desc.metadata['config'] = conf._asdict()
       self.last_run_time_millis = util.time_millis()
 
+      if not any((isinstance(test_start, PhaseDescriptor), test_start is None)):
```

Comment:
any() with only 2 things is kind of weird, probably more readable to use a binary operator and factor the outer `not` through.